api/v1alpha1/oadp_types.go

```diff
@@ -176,10 +176,10 @@ type DataMover struct {
 	Timeout string `json:"timeout,omitempty"`
 	// the number of batched volumeSnapshotBackups that can be inProgress at once, default value is 10
 	// +optional
-	MaxConcurrentBackupVolumes *int64 `json:"maxConcurrentBackupVolumes,omitempty"`
+	MaxConcurrentBackupVolumes string `json:"maxConcurrentBackupVolumes,omitempty"`
```

Comment:
Why string?

Reply:
Was going for consistency: https://github.com/openshift/oadp-operator/blob/master/api/v1alpha1/oadp_types.go#L176

Reply:
Timeout was string because it also have text descriptors.. like 10m vs 10h. Works for me

Reply:
A more convenient reason imo would be to that there is no conversion required in corev1.EnvVar
```
		dataMoverContainer.Env = append(dataMoverContainer.Env, corev1.EnvVar{
			Name:  DataMoverConcurrentBackup,
			Value: dpa.Spec.Features.DataMover.MaxConcurrentBackupVolumes,
		})
```

I would add validation to prevent negative inputs, 0, trailing/prefix whitespace etc.. either via kubebuilder CRD and/or reconciler.